ProcessLib/LIE/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -47,15 +47,16 @@ std::unique_ptr<Process> createHydroMechanicsProcess(
     DBUG("Create HydroMechanicsProcess with LIE.");
 
     // Process variables
+    //! \ogs_file_param{process__HYDRO_MECHANICS_WITH_LIE__process_variables}
     auto const pv_conf = config.getConfigSubtree("process_variables");
-    //! \ogs_file_param_special{process__HYDRO_MECHANICS_WITH_LIE_process_variables__process_variable
-    auto range = pv_conf.getConfigParameterList<std::string>("process_variable");
+    auto range =
+        //! \ogs_file_param{process__HYDRO_MECHANICS_WITH_LIE__process_variables__process_variable}
+        pv_conf.getConfigParameterList<std::string>("process_variable");
     std::vector<std::reference_wrapper<ProcessVariable>> process_variables;
     for (std::string const& pv_name : range)
     {
-        if (pv_name != "pressure"
-            && pv_name != "displacement"
-            && pv_name.find("displacement_jump")==std::string::npos)
+        if (pv_name != "pressure" && pv_name != "displacement" &&
+            pv_name.find("displacement_jump") == std::string::npos)
```

Comment:
Not docu, but I think the last comparison should be `pv_name.find("displacement_jump") != 0`.